test-cases/performance/perf-regression-latency-k8s-multitenant.yaml

```diff
@@ -0,0 +1,30 @@
+# NOTE: need to define following config options in the 'configurations' files:
+# - prepare_write_cmd
+# - stress_cmd_w
+# - stress_cmd_m
+# - stress_cmd_r
+# - n_loaders
+# - k8s_n_loader_pods_per_cluster
+
+test_duration: 480
+user_prefix: 'k8s-perf-regression-latency'
+
+# NOTE: we disable second type of monitoring - K8S based one. It is redundant while we use standalone ones.
+k8s_deploy_monitoring: false
+
+n_db_nodes: 3
+n_monitor_nodes: 1
+
+instance_type_loader: 'c4.2xlarge'
```

Comment:
If we change to i4i, I would also replace these to c5.

Reply:
The same here. Relation to the previous runs to have correct comparison results.